taca/cleanup/cleanup.py

```diff
@@ -211,10 +211,20 @@ def cleanup_irma(days_fastq, days_analysis, only_fastq, only_analysis, status_db
     # make a connection for project db #
     pcon = statusdb.ProjectSummaryConnection(conf=status_db_config)
     assert pcon, "Could not connect to project database in StatusDB"
+    
+    # make exclude project list if provided
+    exclude_list = []
+    if exclude_projects:
```

Comment:
@vezzi have it in the new [commit](https://github.com/SciLifeLab/TACA/pull/218/commits/285886d81ed48cc33585b880129584e0f53fe302). Also checked with an test case 👍 

```
(master)130-229-11-136-dhcp:senthil$ taca cleanup irma --days_fastq 30 --days_analysis 30 --exclude_projects P5358,P275899 -n
2017-02-02 11:00:48,086 - taca.cleanup.cleanup - ERROR - '--exclude_projects' was called with some invalid projects 'P275899', provide valid project name/id
```